src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const isSelected = selectTab.id === tab.id;
+
+            return (
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={cn({
+                  'is-active': isSelected,
+                })}
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => {
+                    if (!isSelected) {
+                      onTabSelected(tab);
+                    }
+                  }}
```

Comment:
Move the onClick logic to a separate function. It follows the best practice of separating concerns and keeping your JSX code clean and focused on rendering components rather than defining complex logic directly within it.